Python/pywarpx/picmi.py

```diff
@@ -923,15 +923,31 @@ class EmbeddedBoundary(picmistandard.base._ClassWithInit):
                  when the solver is electrostatic. Optional, defaults to 0.
      Parameters used in the expressions should be given as additional keyword arguments.
     """
-    def __init__(self, implicit_function, potential=None, **kw):
+    def __init__(self, implicit_function=None, stl_file=None, stl_scale=None, stl_center=None, stl_reverse_normal=None,
+                 potential=None, **kw):
+
+        assert stl_file is None or implicit_function is None, Exception('Only one between implicit_function and '
+                                                                            'stl_file can be specified')
+
         self.implicit_function = implicit_function
+        self.stl_file = stl_file
```

Comment:
Since this is the user-facing interface, please update the doc string of this method:
- list the new option in the list
- potentially start to describe each parameter